test/acceptance/features/steps/openshift.py

```diff
@@ -333,3 +333,13 @@ def search_resource_lst_in_namespace(self, resource_plural, name_pattern, namesp
             return lst
         print('Resource list is empty under namespace - {}'.format(namespace))
         return None
+
+    def get_secret_info_custom_env_var(self, secret_name, namespace):
+        oc_cmd = f'oc get secrets {secret_name}  -n {namespace} -o "jsonpath={{.data.CustomReady}}"'
+        output, exit_code = self.cmd.run(oc_cmd)
+        exit_code | should.be_equal_to(0)
+        return output
+
+    def oc_apply_yaml_file(self, yaml):
+        (output, exit_code) = self.cmd.run("oc apply -f " + yaml)
+        return output
```

Comment:
reuse existing methods, read this file and then use `oc_apply(...)`

Reply:
oc_apply takes in standard input, hence created a function for yaml

Reply:
ok